Add more info to readme (#30)
eucp-project/atlas: README.md

# Atlas of constrained climate projections- EUCP WP2

> [Atlas](https://eucp-project.github.io/atlas/)
> demonstrates some outputs from the probabilistic projection methods, developed
> or assessed in [European Climate Projection system](https://www.eucp-project.eu/) (EUCP)
> related to Work Package 2 (WP2) climate projections. For more info, see the
> [Atlas about page](https://eucp-project.github.io/atlas/about).
> This repository is a collection of maps and scripts to create the Atlas.

Reviewer: ```suggestion
This repository contains the source code and content for an [Atlas of constrained climate projections](https://eucp-project.github.io/atlas/). The atlas demonstrates outputs from the probabilistic projection methods developed or assessed in the [European Climate Projection system](https://www.eucp-project.eu/) (EUCP)
Horizon2020 project. For more info, see the [Atlas about page](https://eucp-project.github.io/atlas/about).
```

## Citation

To cite this repository, use the information avialable at [FIXME](FIXME),
and to cite the content of the Atlas, see the [Atlas references page]FIXME.

## Maintainers

> Current maintainers of the Atlas are some of the Research software engineers from

Reviewer: ```suggestion
Current maintainers of the Atlas are Research software engineers from the
```

[Netherlands eScience Center](https://www.esciencecenter.nl/).
If you have any questions or concerns, please submit an
[issue](https://github.com/eucp-project/atlas/issues). Maintainers will do their

> best to help you

Reviewer: ```suggestion
best to help you.
```

## Contributing

> This section demonstrates instructions for adding maps to Atlas.

Reviewer: ```suggestion
The atlas is created with [Nuxt.js](https://nuxtjs.org/docs/get-started/installation). This section provides instructions for adding maps to Atlas.
```

> 1. Build Setup

Reviewer: ```suggestion
1. Create a local build of the application
```
